spec/models/category_spec.rb

```diff
@@ -5,28 +5,29 @@
   it { should validate_presence_of(:name)}
 
   describe "recent videos" do 
+    before do 
+      @drama = Category.create(name: "Drama")
```

Comment:
As I mentioned in previous comment, I recommend using `let` - that is the right RSpec way.